jwql/tests/test_utils.py

```diff
@@ -344,3 +345,46 @@ def test_filesystem_path():
     location = os.path.join(get_config()['filesystem'], 'jw96003', filename)
 
     assert check == location
+
+
+def test_validate_config():
+    """Test that the config validator works.
+    """
```

Comment:
If the docstring only spans one line, I think we usually just put the description and both`"""` all on one line